create_person.rb

```diff
@@ -17,8 +17,15 @@ def create_person
       name = gets.chomp
       new_student = Student.new(age, classroom)
       new_student.name = name unless name.empty?
-      @people << new_student
-      puts " #{new_student.name} added successfully"
+      student_obj = {
+        Name: new_student.name,
+        Age: new_student.age,
+        Classroom: new_student.classroom,
+        ID: new_student.id,
+        Person: new_student.class,
+        Rentals: []
+      }
+      @people << student_obj
```

Comment:
- [ ] Great job saving and retrieving data. The application saves and works mostly as expected. The only issue comes when I already have some data in the "database" (.json) files on app start and I add some more data, the original data gets overwritten and now only the new data is present in the .json files. Kindly look into this.

[Screencast from 13-09-2022 10:10:37 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/189990208-773ddf05-b7b1-498e-bea5-1a796f068e57.webm)